src/js/components/utils/detailPageLayout.tsx

```diff
@@ -147,12 +147,15 @@ const DetailPageLayout = ({
         </div>
       </div>
       <div
-        className="slds-p-around_x-large
+        // tabIndex required for toast close focus
+        tabIndex={-1}
+        className="metecho-toast-focus
+          slds-p-around_x-large
           slds-grid
           slds-gutters
           slds-wrap"
       >
-        <div
+        <aside
```

Comment:
Should this `<aside>` node also have a role, or are a11y roles implicit in this tag? I found [this page](https://www.w3.org/WAI/GL/wiki/Using_HTML5_aside_element) that says:

> adding the ARIA role="note" as follows or role="complementary" is advised.

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role the "complimentary" landmark role is not needed, as it it implied with "<aside>".

Reply:
I'm also noticed on the page you linked it says "If support for HTML5 within a user agent is in doubt, then add the ARIA role="complementary"." I don't think we need to worry about HTML5 support.